frontend/encore/bootstrap.rst

```diff
@@ -1,18 +1,24 @@
-Using Bootstrap CSS & JS
-========================
+Using Bootstrap 4 CSS & JS
+============================
 
 Want to use Bootstrap (or something similar) in your project? No problem!
 First, install it. To be able to customize things further, we'll install
-``bootstrap-sass``:
+``bootstrap`` and popper.js:
```

Comment:
Let's revert this change because the article is about Bootstrap, so we don't want to install other libraries. Thanks!

Reply:
I delete popper.js but bootstrap 4 need it to work with tooltips. (Yarn and nmp say to add this dependancie when we install bootstrap so is ok)